hack/testing/entrypoint.sh

```diff
@@ -22,16 +22,19 @@ source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
 source "${OS_O_A_L_DIR}/hack/testing/util.sh"
 
 # HACK HACK HACK
-# There seems to be some sort of performance problem - richm 2017-08-15
-# not sure what has changed, but now running an all-in-one for CI, with both
-# openshift master and node running as systemd services logging to the journal
-#, and the default/logging pods, and the os, are spewing too much for fluentd
-# to keep up with when it has 100m cpu (default), on a aws m4.xlarge system
-# for now, remove the limits on fluentd to unblock the tests
+#
+# There seems to be some sort of performance problem - richm 2017-08-15 not
+# sure what has changed, but now running an all-in-one for CI, with both
+# openshift master and node running as systemd services logging to the
+# journal, and the default/logging pods, and the os, are spewing too much for
+# fluentd to keep up with when it has 100m cpu (default), on a aws m4.xlarge
+# system for now, remove the limits on fluentd to unblock the tests
 oc get -n logging daemonset/logging-fluentd -o yaml > "${ARTIFACT_DIR}/logging-fluentd-orig.yaml"
 if [ -z "${USE_DEFAULT_FLUENTD_CPU_LIMIT:-}" ] ; then
     oc patch -n logging daemonset/logging-fluentd --type=json --patch '[
-          {"op":"remove","path":"/spec/template/spec/containers/0/resources/limits/cpu"}]'
+          {"op":"add","path":"/spec/template/spec/containers/0/resources/limits/cpu", "value": "1000m"}]'
+    oc patch -n logging daemonset/logging-fluentd --type=json --patch '[
+          {"op":"add","path":"/spec/template/spec/containers/0/resources/requests/cpu", "value": "1m"}]'
```

Comment:
@richm, @jcantrill, I am not sure why this works, but it seems to.  My guess is that the very low "requests" value allows the fluentd process to be scheduled on whatever amount of CPU is available, and the 1 CPU limits value gives it enough range to actually work within the 300s timeout window.